packages/admin-panel/package.json

```diff
@@ -27,59 +27,61 @@
     "start-entity-server": "yarn workspace @tupaia/entity-server start-dev",
     "start-report-server": "yarn workspace @tupaia/report-server start-dev",
     "start-admin-panel-server": "yarn workspace @tupaia/admin-panel-server start-dev",
+    "start-data-table-server": "yarn workspace @tupaia/data-table-server start-dev",
     "test": "echo No tests",
     "start-ui-components": "yarn workspace @tupaia/ui-components build-dev:watch",
     "start-frontend": "npm-run-all -c -l -p start-ui-components start-dev",
-    "start-servers": "npm-run-all -c -l -p start-central-server start-entity-server start-report-server start-admin-panel-server"
+    "start-servers": "npm-run-all -c -l -p start-central-server start-entity-server start-report-server start-admin-panel-server start-data-table-server"
   },
   "browserslist": [
     "defaults"
   ],
   "dependencies": {
-    "@material-ui/core": "^4.9.11",
-    "@material-ui/icons": "^4.9.1",
-    "@material-ui/lab": "^4.0.0-alpha.47",
-    "@material-ui/styles": "^4.9.10",
+    "@material-ui/core": "^4.12.4",
+    "@material-ui/icons": "^4.11.3",
+    "@material-ui/lab": "^4.0.0-alpha.61",
+    "@material-ui/styles": "^4.11.5",
     "@tupaia/access-policy": "3.0.0",
     "@tupaia/types": "1.0.0",
     "@tupaia/ui-components": "1.0",
     "@tupaia/utils": "1.0.0",
-    "axios": "^0.21.1",
-    "case": "^1.5.3",
+    "axios": "^0.21.4",
+    "case": "^1.6.3",
     "content-disposition-header": "^0.6.0",
-    "date-fns": "^2.29.2",
+    "date-fns": "^2.29.3",
     "file-saver": "^1.3.3",
-    "localforage": "^1.5.0",
+    "localforage": "^1.10.0",
     "lodash.debounce": "^4.0.8",
     "lodash.keyby": "^4.6.0",
     "lodash.throttle": "^4.1.1",
-    "moment": "^2.22.2",
-    "parse-link-header": "^1.0.1",
-    "prop-types": "^15.6.0",
+    "moment": "^2.29.4",
+    "parse-link-header": "^2.0.0",
+    "prop-types": "^15.8.1",
     "query-string": "^5.1.1",
-    "react": "^16.13.1",
+    "react": "17.0.0",
     "react-autobind": "^1.0.6",
     "react-beforeunload": "^2.5.1",
-    "react-datepicker": "^1.6.0",
-    "react-dom": "^16.13.1",
-    "react-hook-form": "^6.8.3",
+    "react-datepicker": "^1.8.0",
+    "react-dom": "17.0.0",
+    "react-hook-form": "^6.15.8",
     "react-icons": "^2.2.7",
-    "react-password-strength-bar": "^0.3.2",
-    "react-query": "^3.19.0",
-    "react-redux": "^5.0.6",
-    "react-router-dom": "^5.2.0",
-    "react-select": "^1.2.1",
-    "react-table": "^6.7.6",
+    "react-password-strength-bar": "^0.3.5",
+    "react-query": "^3.39.3",
+    "react-redux": "^5.1.2",
+    "react-router-dom": "^5.3.4",
+    "react-select": "^1.3.0",
+    "react-table": "^6.11.5",
```

Comment:
Due to the advanced MUI component DataTable requiring minimum react 17.0.0, also needed to upgrade packages with react dependency, including react-dom. Needed to do this for all ui packages with ui-components as a dependency.